packages/web-ui/src/ds/atoms/Illustrations/Evaluation.tsx

```diff
@@ -0,0 +1,167 @@
+import { LucideProps } from 'lucide-react'
+
+export default function LatitudeLogo(props: LucideProps) {
```

Comment:
Wrong name

Reply:
Should be: `export default function EvaluationIllustration() {` ?